src/main/java/soolutions/hometaskapp/basic/BasicAccount.java

```diff
@@ -0,0 +1,19 @@
+package soolutions.hometaskapp.basic;
+
+import soolutions.hometaskapp.Account;
+import soolutions.hometaskapp.User;
+
+public class BasicAccount implements Account {
+    public BasicAccount(User user) {
+    }
+
+    @Override
+    public boolean open() {
+        return true;
+    }
+
+    @Override
+    public int balance() {
+        return 0;
```

Comment:
What happens if other methods are called before `open`?

Reply:
Indeed, an instance of `Account` should already be ready to be queried. Perhaps what we need is an `Accounts.open()` that returns a valid `Account`.

Reply:
Why balance is `int`? it should be `double` or `float` imho. Also, I would suggest to have an `Amount` with value and currency.